### PR TITLE
Add missing note

### PR DESCRIPTION
**Pull request by KevinVlaanderen**
After discussion with Jonathan, this note should be added after all (after earlier discussion with Adam, in which it was decided not to add it)

### CODE REVIEW

File: content/releasenotes/studio-pro/9.11.md [hunk 1]
```diff
@@ -32,6 +32,7 @@ weight: 90
 * We added support to autocomplete XPath and OQL with the <kbd>Tab</kbd> key.
 * We improved security by removing microflow information from *metamodel.json*. Microflow information is now only available in the first session request (`get_session_data`) and only contains the microflows to which the user has access.
 * We improved the error message that was shown when trying to use a [Commit object](/refguide/committing-objects) activity on an external object. A suggestion is added to use the [Send external object](/refguide/send-external-object) activity instead, whenever the external entity allows for updates.
+* We added a new setting to specify the ‘cost’ of the bcrypt hash algorithm. This can be a value from 10 (the default) to 30.
```
Comment (Adam-Dupaski) [inline content/releasenotes/studio-pro/9.11.md:35]: How does this differ from the note directly below?

Comment (KevinVlaanderen) [reply content/releasenotes/studio-pro/9.11.md:35]: The latter is about the ability to specify the cost in part of a custom setting; the former, new one is about the setting in Studio Pro

Comment (Adam-Dupaski) [reply content/releasenotes/studio-pro/9.11.md]: Thanks, clear

**Review by Adam-Dupaski — APPROVED** [review]

**Comment by Adam-Dupaski** [comment]
CC @NatasaKralj @MarkvanMents

**Pull request merged**